content/en/apps/reference/api.md

```diff
@@ -813,6 +817,7 @@ Content-Type: application/json; charset=utf-8
     "contact": {
       "_id": "eeb17d6d-5dde-c2c0-62c4a1a0ca17fd17",
       "type": "person",
+      "roles": [ "chw" ]
```

Comment:
I'm unsure if the "roles" property needs to be added to the contact.

Reply:
Definitely not! Fixed, thanks.